db/migrate/20151117102656_create_reviews.rb

```diff
@@ -0,0 +1,11 @@
+class CreateReviews < ActiveRecord::Migration
+  def change
+    create_table :reviews do |t|
+      t.integer :video_id
+      t.integer :rating
+      t.string :author
+      t.text :comment
```

Comment:
I'm not a fan of this name. The *thing* here is a review. This name should reflect what portion of the review this element is. Comment sounds more like an object in its own right.